content/naorlivne/drone-metronome/index.md

```diff
@@ -0,0 +1,76 @@
+---
+version: '1.0'
+date: 2019-08-27T00:00:00+00:00
+title: Metronome
+author: naorlivne
+tags: [ deploy, metronome ]
+logo: metronome.svg
+repo: naorlivne/drone-metronome
+image: naorlivne/drone-metronome
+---
+
+This plugin can be used to deploy applications to a Metronome server, it will create\update the given Metronome tasks as needed.
+
+The below pipeline configuration demonstrates simple usage:
+
+```yaml
+pipeline:
```

Comment:
This is not an valid example.

Reply:
I assume the you referring to using the old 'pipeline:'  format instead of the new 'kind: pipeline' format, which is fixed now... if you meant something else please be more specific.

Reply:
Now the examples are fine, I was referring to a valid 1.0.0 example :)